node/src/accountant/mod.rs

```diff
@@ -3319,12 +3328,12 @@ mod tests {
         pp_logger
     }
 
-    fn assert_payable_scanner_for_no_p_p_found(
+    fn assert_payable_scanner_for_no_pending_payable_found(
         payable_scanner_start_scan_arc: &Arc<
             Mutex<Vec<(Wallet, SystemTime, Option<ResponseSkeleton>, Logger, String)>>,
         >,
         notify_and_notify_later_params: &NotifyAndNotifyLaterParams,
-        compute_interval_until_next_new_payable_scan_params_arc: Arc<Mutex<Vec<()>>>,
+        time_until_next_scan_until_next_new_payable_scan_params_arc: Arc<Mutex<Vec<()>>>,
```

Comment:
You only need, `time_until_next_new_payable_scan_params_arc`, there's a repetition in the name... `next` and `scan`.